WireDomain/Sources/WireDomain/Account/AccountStore.swift

```diff
@@ -154,6 +154,15 @@ struct AccountStore {
         }
     }
 
+    func mainWebsocketLockFileURL(accountID: UUID) throws -> URL {
```

Comment:
todo: to remove

Reply:
done https://github.com/wireapp/wire-ios/pull/3379/commits/e445fae23796acf097debd88dabdd568d652add3